packages/commonwealth/server/routes/communities/put_community_stakes_handler.ts

```diff
@@ -23,11 +23,19 @@ export const putCommunityStakeHandler = async (
     throw new AppError(formatErrorPretty(paramsValidationResult));
   }
 
-  await validateCommunityStakeConfig(
-    models,
-    paramsValidationResult.data.community_id,
-    paramsValidationResult.data.stake_id,
-  );
+  const { community_id, stake_id } = paramsValidationResult.data;
+
+  const adminAddress = (await req.user.getAddresses())
```

Comment:
For testing purposes, we should allow superadmin access. We might as well use the official utility to perform the check succinctly:
```ts
  const isAdmin = await validateOwner({
    models: this.models,
    user,
    communityId: community.id,
    allowMod: true, // optional
    allowAdmin: true,
    allowGodMode: true,
  });
```